src/docs/api/public.mdx

```diff
@@ -182,24 +184,23 @@ class TeamExamples:
 
 See [here](https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_teams.py) for an example of an endpoint documented using drf-spectacular, and the [drf-spectacular docs here](https://drf-spectacular.readthedocs.io/en/latest/) to learn more.
 
-**Note that if the endpoint you're modifying previously had JSON documentation, you will need to
-delete the old documentation path found in [this
-file](https://github.com/getsentry/sentry/blob/master/api-docs/openapi.json) along with its
-corresponding JSON build in [this
-folder](https://github.com/getsentry/sentry/blob/master/api-docs/paths).**
+**Note that if the endpoint you're modifying had previous JSON documentation, you must delete the 
+old documentation path in [this file](https://github.com/getsentry/sentry/blob/master/api-docs/openapi.json) 
+and its corresponding JSON build in [this folder](https://github.com/getsentry/sentry/blob/master/api-docs/paths).**
 
 **Additionally if there are multiple request types in the same endpoint using the old JSON
 documentation, you must update both of them in the same PR. Updating only one request and
-deleting the documentation path/JSON will cause the other unedited request to disappear from the
-docs.**
+deleting the old documentation will cause the other unedited request to disappear from the docs.**
 
 **Tips**:
 
 - `make test-api-docs` builds the OpenAPI JSON and runs all API docs tests.
 - To only build the docs, run `make build-api-docs`. The build will fail if there are any warnings.
 - To see the diff of your newly generated docs with prod, run `make diff-api-docs`. This requires you to have built the OpenAPI JSON already.
 
-**To see your changes in the docs locally**:
+### Building locally
```

Comment:
I don't know why this line is causing vercel to fail :sweat: I tried rendering the mdx locally and it worked fine